doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,351 @@
+pulls the required image. This can take some time, depending on the size of the image.
+
+To rerun it, you should restart the container with this command:
+
+.. code:: bash
+
+   docker start mapdl
+
+Or you can delete the container and run it again using these commands:
+
+.. code:: bash
+
+    docker rm -f mapdl
+
+    docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $LOCAL_MAPDL_PORT:50052   $MAPDL_DOCKER_REGISTRY_URL -smp > log.txt
+
+
+You can append the Docker flag ``--rm`` to automatically clean up the container
+when it exits.
+
+The preceding commands create a log file (``log.txt``) in your current directory location.
+However, you can remove ``> log.txt `` if you don't want to create this file. In this case,
+the command output is redirected to the console, which is kept blocked until the Docker
+image exits. You can detach the console from the Docker container output by appending
+``-d`` to the `docker run <docker_run_>`_  command. (Always add this before the Docker
+image URL.)
+
+If you don't want to block the console, the best approach is to pipe the output to a file
+as mentioned earlier so that you can inspect the output of that file.
+
+Notice that the MAPDL Docker image gRPC port (``50052``) is being mapped to a
+different host port (``50053``) to avoid port conflicts with local
+MAPDL instances running on the host or other Docker images.
+You could additionally launch more Docker containers in different ports if
+you want to run multiple simulations at the same time.
+
+The :ref:`ref_pymapdl_pool`` module does not work when you are connecting
+to a remote MAPDL Docker image. It does not work when connected to Docker
+containers either. If you decide to launch multiple MAPDL instances, you must
+manage these instances yourself.
+
+.. note:: Ensure that port ``50053`` is open in your local firewall.
+
+You can provide additional MAPDL command line parameters to MAPDL by simply
+appending them to the end of the command.
+
+For example, you can increase the number of processors (up to the
+number available on the host machine) with the ``-np`` switch:
+
+.. code:: bash
+
+  docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always -d --name mapdl -p $LOCAL_MAPDL_PORT:50052 $MAPDL_DOCKER_REGISTRY_URL -smp -np 8 > log.txt
+
+
+For additional command line arguments, see the *Notes* section in the
+description for the :func:`launch_mapdl() <ansys.mapdl.core.launch_mapdl>`
+function.
+
+You can use a script file (batch ``'.bat'`` or PowerShell ``'.ps'``)
+to run the preceding commands all at once.
+
+Once you have launched MAPDL, you should see the following content
+in your console (or the output file):
+
+.. code::
+
+    Start GRPC Server
+
+    ##############################
+    ### START GRPC SERVER      ###
+    ##############################
+
+    Server Executable   : MapdlGrpc Server
+    Server listening on : 0.0.0.0:50052
+
+
+.. warning:: 
```

Comment:
This should probably be a note. See this [style guide topic](https://developers.google.com/style/notices).

Reply:
Good idea! Thank you!